README.md

```diff
@@ -46,19 +46,19 @@ from __future__ import print_function
 
 import squareconnect
 from squareconnect.rest import ApiException
-from squareconnect.apis.location_api import LocationApi
+from squareconnect.apis.locations_api import LocationsApi
 
 # setup authorization
 squareconnect.configuration.access_token = 'YOUR_ACCESS_TOKEN'
 # create an instance of the Location API class
-api_instance = LocationApi()
+api_instance = LocationsApi()
 
 try:
     # ListLocations
     api_response = api_instance.list_locations()
     print (api_response.locations)
 except ApiException as e:
-    print ('Exception when calling LocationApi->list_locations: %s\n' % e)
```

Comment:
Not quite sure if that's the correct syntax. 

I specifically just changed word "LocationApi" to "LocationsApi", so I just added the letter "s" and made it plural. README.md instructions are working fine after this.